src/api/app/models/token/workflow.rb

```diff
@@ -42,8 +42,8 @@ def call(options)
   end
 
   def owned_by?(some_user)
-    # TODO: remove the first condition if we migrate the Token.user to Token.users
-    user == some_user || users.include?(some_user) || groups.map(&:users).flatten.include?(some_user)
+    # TODO: remove the first condition if we migrate the Token.user to Token.users_shared_among
```

Comment:
`users_shared_among` doesn't exist anymore, it was renamed to `users`. The first part of the comment could be clearer and `Token.user` is now `Token.executor`:

```suggestion
    # TODO: remove the first condition if we data-migrate the Token.executor to Token.users
```